conf/messages

```diff
@@ -862,6 +862,13 @@ benefits.medicalDentalChildcare.title.agent = Did your client get any medical, d
 benefits.medicalDentalChildcare.error.noEntry.individual = Select yes if you got medical, dental, childcare, education benefits or loans
 benefits.medicalDentalChildcare.error.noEntry.agent = Select yes if your client got medical, dental, childcare, education benefits or loans
 
+benefits.medicalDentalBenefits.title.individual = Did you get a medical or dental benefit?
+benefits.medicalDentalBenefits.title.agent = Did your client get a medical or dental benefit?
+benefits.medicalDentalBenefits.content.individual = This is medical or dental treatment or insurance provided by your employer.
+benefits.medicalDentalBenefits.content.agent = This is medical or dental treatment or insurance provided by your client’s employer.
```

Comment:
This message is not the same as in the requirements

Reply:
I copied it from the prototype, is that not the right place ?

Reply:
You should get it from here: https://docs.google.com/presentation/d/1ZK243ozKJdAkB_6veZAKBJ6OGEoIt2EW1IXkDmEW_h4/edit#slide=id.gd67e1cf737_0_202

Reply:
Ok, thaks